library/core/src/prelude/common.rs

```diff
@@ -79,6 +79,15 @@ pub use crate::macros::builtin::{
 #[unstable(feature = "derive_const", issue = "none")]
 pub use crate::macros::builtin::derive_const;
 
+/*
+#[cfg(not(bootstrap))]
+#[unstable(feature = "rustc_contracts", issue = "none")]
+pub use crate::macros::builtin::{rustc_contracts_requires, rustc_contracts_ensures};
+*/
+#[cfg(not(bootstrap))]
+#[unstable(feature = "rustc_contracts", issue = "none")]
+pub use crate::contracts;
```

Comment:
Adding experimental stuff to the prelude is probably premature.

Reply:
yeah that's probably true.

(context: an earlier version of this branch had named this module `rustc_contracts` instead of `contracts`, in order to enable `#[rustc_contract::requires(...)`. but for some reason our handling of `rustc_`-prefixed attributes made that not work at all, and so I fell back on this approach. But you are right that it doesn't need to be in the prelude.)

Reply:
yes I'll remove the change to the prelude.